sklearn/datasets/_samples_generator.py

```diff
@@ -1609,6 +1614,12 @@ def make_sparse_spd_matrix(
     largest_coef : float, default=0.9
         The value of the largest coefficient between 0 and 1.
 
+    sparse_format : str, default=None
+        String[s] representing the output sparse format, such as 'csc', 'csr', etc.
```

Comment:
We can only pass one sparse format at a time, so the singular is fine, no?

```suggestion
        String representing the output sparse format, such as 'csc', 'csr', etc.
```

Reply:
True it should be singular. I copied from the `accept_sparse` without a second thought.